src/util/ProjectPropertiesExtractor.ts

```diff
@@ -0,0 +1,33 @@
+
+import {LocalProject} from "@atomist/automation-client/project/local/LocalProject";
+import * as child_process from "child_process";
+import * as properties from "properties";
+
+export interface ProjectProperties {
+    group: string;
+    name: string;
+    version: string;
+}
+
+export class ProjectPropertiesExtractor {
```

Comment:
These methods should probably all be async and return promises.

Reply:
Maybe, but I'd be interested in your reasoning for it.
Check out the usage of it in IdentifyProjectOnCommit, it is wrapped in a promise due to the usage of GitCommandGitProject.